README.org

```diff
@@ -0,0 +1,81 @@
+* envlookup
+
+[[https://travis-ci.org/thepatrik/envlookup][https://travis-ci.org/thepatrik/envlookup.svg?branch=master]]
```

Comment:
It would be nice if travis was configured for the spider-pigs org. Then we could point this badge url to that instead.